staging/src/k8s.io/component-base/go.mod

```diff
@@ -86,5 +87,8 @@ require (
 replace (
 	k8s.io/api => ../api
 	k8s.io/apimachinery => ../apimachinery
+	k8s.io/apiserver => ../apiserver
 	k8s.io/client-go => ../client-go
+	k8s.io/component-base => ../component-base
```

Comment:
this seems a cyclic dependency ... the job to test dependencies also reports it as a problem, hack the `hack/update-vendor.sh` script

Reply:
Resolved now.